### DISCUSSION TITLE
Without label file

### DISCUSSION THREAD

**Discussion by Nithishrish23** [ultralytics/ultralytics#6109] [Q&A]
@glenn-jocher

 Is it possible to train a data without labels, I had only required images without extra background. Is it works on YOLO Model 
shall i give only images without labels is it possible understand a Yolo without annotations with only images .Is it Works or not

**Comment by glenn-jocher** [comment]
@Nithishrish23 hello! 👋

Training a YOLO model typically requires both images and corresponding label files that provide annotations for the objects within those images. The annotations are essential as they tell the model what objects are present and where they are located in the image.

Without label files, you cannot perform supervised learning, which is what YOLO models are designed for. However, if you're interested in unsupervised learning or feature extraction without labels, you might want to explore other types of models or techniques that don't rely on annotated datasets.

For YOLO and other supervised object detection models, having well-annotated data is crucial for effective training and performance. If you need assistance with labeling your images, there are tools and services available that can help streamline the annotation process.

If you have any more questions or need further assistance, feel free to ask! 🚀
